packages/universal-router-sdk/test/utils/permit.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  AllowanceTransfer,
+  PermitSingle,
+  MaxAllowanceExpiration,
+  MaxAllowanceTransferAmount,
+} from '@pancakeswap/permit2-sdk'
+import { Hex, type Address, type WalletClient, hexToSignature, toBytes, concat, toHex } from 'viem'
+
+const TEST_DEADLINE = MaxAllowanceExpiration
+
+export const makePermit = (
+  token: Address,
+  // as spender
+  routerAddress: Address,
+  amount: string = MaxAllowanceTransferAmount.toString(),
+  nonce = 0,
+): PermitSingle => {
+  return {
+    details: {
+      token,
+      amount,
+      expiration: TEST_DEADLINE,
+      nonce,
+    },
+    spender: routerAddress,
+    sigDeadline: TEST_DEADLINE,
+  }
+}
+
+export const signPermit = async (permit: PermitSingle, wallet: WalletClient, permit2Address: Address): Promise<Hex> => {
```

Comment:
Let's keep the unit test within its own scope. For unit test for universal router, we can use a arbitrary wallet and generate the signature and use it as a constant literal

Reply:
So no need to actually sign the permit, we can simply use the signature as a constant